verticapy/core/vdataframe/_fill.py

```diff
@@ -453,7 +456,7 @@ def fillna(
                         sql_push_ext=self._parent._vars["sql_push_ext"],
                         symbol=self._parent._vars["symbol"],
                     )
-                except:
+                except QueryError:
```

Comment:
You are importing it as vQueryError

Reply:
there are two different QueryError one from VerticaPy and one from Vertica-Python.